docs-src/src/pages/design-tokens.js

```diff
@@ -170,17 +158,253 @@ function Fonts() {
-            <Table.Header>
-              Token
-            </Table.Header>
-            <Table.Header>
-              Value
-            </Table.Header>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
           </Table.Row>
         </Table.Head>
-        <Table.Body>
-          {FontRows()}
-        </Table.Body>
+        <Table.Body>{FontRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+function BorderRadiusRows() {
+
+  const rows = []
+  const radii = DesignTokens.border.radius
+
+  for (const radius in radii) {
+    const exampleStyle = {
+      borderRadius: radii[radius].value,
+      width: '10rem',
+      height: '3rem',
+      border: '1px solid black',
+    };
+
+    rows.push (
+    <Table.Row>
+      <Table.Data>
+        <Table.HeaderInline>Token:</Table.HeaderInline>
+          <pre>${radii[radius].name}</pre>
+          <p><i>{radii[radius].comment}</i></p>
+      </Table.Data>
+      <Table.Data>
+        <Table.HeaderInline>Value:</Table.HeaderInline>
+        <div style={exampleStyle}></div>
+        <pre>{radii[radius].value}</pre>
+      </Table.Data>
+    </Table.Row>
+    )
+  }
+
+  return rows
+
+}
+
+function BorderRadius() {
+  return (
+    <React.Fragment>
+      <h2>Border Radius</h2>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>{BorderRadiusRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+// TODO is exampleStyle necessary here?  How can I eliminate this useless thang
+function ZIndexRows() {
+  
+  const rows = []
+  const zIndices = DesignTokens["z-index"]
+
+  for (const index in zIndices) {
+    const exampleStyle = {
+      zIndex: zIndices[index].value
+    };
+
+    rows.push (
+      <Table.Row>
+        <Table.Data>
+          <Table.HeaderInline>Token:</Table.HeaderInline>
+            <pre>${zIndices[index].name}</pre>
+            <p><i>{zIndices[index].comment}</i></p>
+        </Table.Data>
+        <Table.Data>
+          <Table.HeaderInline>Value:</Table.HeaderInline>
+          <pre>${zIndices[index].value}</pre>
+        </Table.Data>
+      </Table.Row>
+      )
+    }
+
+    return rows
+}
+
+function ZIndex() {
+  return (
+    <React.Fragment>
+      <h2>Z-Index</h2>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            <Table.Header>Token</Table.Header>
+            <Table.Header>Value</Table.Header>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>{ZIndexRows()}</Table.Body>
+      </Table>
+    </React.Fragment>
+  )
+}
+
+
+// TODO fix name not being displayed at same spacing as value!
+function SpacingRows() {
+
+  const rows = []
+  const spaces = DesignTokens.spacing
+
+  for (const space in spaces) {
+    const exampleStyle = {
+      spacing: spaces[space].value,
```

Comment:
This line doesn't do anything so you can remove it

Reply:
Done